lib/src/l10n/app_en.arb

```diff
@@ -94,7 +94,8 @@
   "ok": "OK",
   "olderVersions": "Older Versions",
   "reboot": "Reboot",
-  "rebootConfirm": "The update requires a reboot to complete. Do you want to reboot now?",
+  "rebootConfirmMessage": "The update requires a reboot to complete. Please save any unsaved work before proceeding. Do you want to reboot now?",
+  "rebootConfirmTitle": "Reboot required",
   "reinstall": "Reinstall",
```

Comment:
GNOME shell uses the terminology 'Restart' so we should adopt that for consistency. If we need to revisit reboot vs restart (or update and upgrade) thats one for user research.